test/templates/icon/component.njk

```diff
@@ -4,5 +4,5 @@
     instead of more complex modifier.block options
 #}
 {%- macro icon(glyph="", size="", classes="") -%}
-<span class="tn-icon{{ (glyph if size == "" else [glyph, size]) | modifier('icon') }}{{ classes | classes }}" role="presentation"></span>
```

Comment:
This should stay. I added this for the dropdown based on what we had discussed. Removing it breaks the dropdown component.

Reply:
this is adding an empty class "tn-"

<span class="tn-icon tn-icon--add tn-" role="presentation"></span>

Reply:
This should be fixed with #978 — https://github.com/SAP/techne/pull/978/files#diff-ace3b50701c30da220db72a852e66f79R52

Reply:
ok will restore and merge.